backend/src/nodes/nodes/image_filter/high_pass.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput
+from ...properties.outputs import ImageOutput
+from . import category as ImageFilterCategory
+
+
+@NodeFactory.register("chainner:image:high_pass")
+class HighPassFilterNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply High Pass filter on image"
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Radius", minimum=0, default=1, precision=2, controls_step=1),
```

Comment:
I think it would be nice if we could use a default value that will produce okay-ish results for most people.

```suggestion
            NumberInput("Radius", minimum=0, default=3, precision=2, controls_step=1),
```